interview-questions.md

```diff
@@ -9,49 +9,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The operators that will return a Boolean value in javascript are true,false, ===, ==,!=, >, <, >=, <=
 
 Researched answer:
+The javascript operators that will return a Boolean value are useful because they allow us to compare data and then execute based on the output of that comparison.The opertatos that return true or false (Boolean) are; relational (<,>,<=,>=), Equality (strict ===, loose ==), and Logical (&& , ||, and !).
+3. What is an index? What is the difference between index and value? 
 
-3. What is an index? What is the difference between index and value?
+Your answer: An index is location or position of a data type in a array. The index is the location and the value is the data stored at that particular index.
 
-Your answer:
-
-Researched answer:
+Researched answer
+ In JavaScript an index is adddress to contain the location of data in an array or string. Value is the data at a given index in an array. We often use this to check what data is being stored and logged in the correct place. We can also find where data has been stored which would be helpful if we need to retrieve that data.
 
 4. What is iteration?
 
-Your answer:
+Your answer: It is repeating a process until we are given instructions to stop. Loops iterate through a set of data until it finds a true statement.
 
-Researched answer:
+Researched answer: Iterations is repeating a proccess a certain number of times. The iterator in for loops tells us what direction we are going through the data positive or negative.
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: Pair programming is done using git and github via the terminal. This is helpful because it gives an extra pair of eyes to code through challenges or problems. The person doing the coding and  is called the driver. This person will pull the repo and clone it down, create a new branch and code. The navigator is the other person watching what the driver is coding and giving instructions as to what code is being input. The navigator does not code they only dicatate so the driver can input. This makes it more efficient as one person can focus on the task while the other is focused on the code.what
 
-Researched answer:
+Researched answer: Pair programming involves two people working on one source code collaboratively to make one program. The idea is that two heads are better than one. When dealing with complex problems two people can pool together their knowledhe base to make better or faster solutions. 
 
 ## Looking Ahead: Terms for Next Week
 
```

Comment:
It's always great to look ahead at the terms so you have some context when hearing the terms during lectures.